### ISSUE TITLE
Mismatch of list items in terms of services 

### ISSUE THREAD

**Issue by Malavi1** [CircuitVerse/CircuitVerse#4626]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

In terms of services we are displaying all the list items as ordered but this is missing in the **Digital Millennium Copyright Act (DMCA)**

![Screenshot 2024-01-20 173911](https://github.com/CircuitVerse/CircuitVerse/assets/112646623/45e66f28-cc19-425a-9f39-4d2b44a604ad)

### Steps To Reproduce

about->terms of service

### Expected Behavior

We have to make that list as a ordered one too.

### Device Information [optional]

```markdown
- OS:
- Browser:
- version:
```

### Are you working on this issue?

No

**Comment by foad-khan** [comment]
Hello! @Malavi1, is this will be a write approach?
>config ->locals -> views -> cv -> en.yml -> DMCA ->changing list as in image(list_item1: to list1_item1) "in all language file .yml"
![Screenshot (77)](https://github.com/CircuitVerse/CircuitVerse/assets/156365622/6f50104d-2425-4e2e-98b8-707f95a3b0e9)

**Comment by AyushRakwal** [comment]
can i do this if not done yet?

**Comment by subhamkumarr** [comment]
@Malavi1, SO what is the correct me if I am wrong?
In the Digital Millennium Copyright Act (DMCA), there should be 3.1 & 3.4  points. Which is present in the Rules of usage?

**Comment by Malavi1** [comment]
No, both 3.1 & 3.4 list contents are ordered except 6.2 have to make ordered as well

**Comment by foad-khan** [comment]
@Malavi1 what about my approach?

**Comment by Malavi1** [comment]
unless it makes any reasonable impact it's not to good to proceed IG.
cc : @tanmoysrt

**Comment by foad-khan** [comment]
Sure! @Malavi1
